File: 3-4/main.py
import sys
import numpy as np


def reverse_a_number(arg):
    num = []
    for i in arg:
        # １文字ずつ配列に格納
        num.append(i)
        # 配列を反転
        renum = num[::-1]
    # 配列の形を崩す
    result = ''.join(renum)
    # 数値へ変換
    result = int(result)
    return(result)


def main():
    # 入力は文字列扱い
    a = sys.argv[1]
    b = sys.argv[2]

    results = reverse_a_number(a) + reverse_a_number(b)
    print(results)


if __name__ == '__main__':
    main()

File: 3-4/main.js
const arga = process.argv[2];
const argb = process.argv[3];

function reverse_a_number(arg){
    //文字のまま受け取り 分割 反転 結合 します
    return arg.split('').reverse().join('');
}

const testResult = reverse_a_number(arga);

const result = Number(reverse_a_number(arga)) 
                + Number(reverse_a_number(argb));


console.log(typeof testResult)
console.log(testResult)

console.log(typeof result)
console.log(result)